packages/block/src/header.ts

```diff
@@ -65,6 +65,16 @@ export class BlockHeader {
     return this.logsBloom
   }
 
+  /**
+   * EIP-4399: Post-PoS merge, `mixHash` supplanted as `random`
+   */
+  get random() {
+    if (!this._common.isActivatedEIP(4399)) {
+      throw new Error('random can only be accessed when EIP-4399 is activated')
```

Comment:
Can you prepare the error msg with the new `_errorMsg()` function (like: `const msg = this._errorMsg(`random can only be accessed when EIP-4399 is activated${errorMsg}`).

Also, this reads a bit ambiguous and hard to grasp what random refers to, maybe `The random parameter can...`.

Reply:
yep for sure, will add, thanks